src/evaluate.jl

```diff
@@ -100,6 +126,25 @@ function evaluate!{T<:Number}(x::Array{TaylorN{T},1}, δx::Array{T,1},
     end
     nothing
 end
+
```

Comment:
I think the two methods below can be written as one, imposing `T<:NumberNotSeriesN`.

Reply:
Thanks @lbenet!